pkgs/applications/editors/emacs/macport.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, ncurses, pkgconfig, texinfo, libxml2, gnutls, gettext, autoconf, automake
+{ stdenv, fetchurl, ncurses, pkgconfig, texinfo, libxml2, gnutls, gettext, autoconf, automake, jansson
```

Comment:
jansson should go in buildInputs

Reply:
Argh, not sure how I missed that out in this PR. Have updated it, cheers 👍